### ISSUE TITLE
Support for google.rpc.Status

### ISSUE THREAD

**Issue by seime** [uw-labs/bloomrpc#30]
Adding a proto file with 

```
import "google/rpc/status.proto";
```

and
```
google.rpc.Status error_response = 99;
```

givers error message

Error while importing protos
no such Type or Enum 'google.rpc.Status' in Type .xxx.xxxx.TypeName

**Comment by seime** [comment]
https://github.com/googleapis/googleapis/tree/master/google/rpc

**Comment by fenos** [comment]
Hi @seime,

This protofiles are very specific to googleapis and not as generic as the **well-known** types such as:
 
- google.protobuf.Empty
- google.protobuf.Timestamp
- google.protobuf.Any 
- more...

everything under `google/protobuf/*` is supported out of the box.

If you are on version `1.2.0` of bloomrpc you can use `import paths` feature where you can provide a local path where those protofiles can be found.

Keep in mind that the path needs to be absolute to your file system but the import will be then relative to that path.

Let me know what are your reasons to have `google/rpc` baked in bloomrpc.

**Comment by seime** [comment]
@fenos , thanks for a swift reply!

The file is included by default in the grpc stack. The link I posted was to the original; this one tells the full story: https://github.com/grpc/grpc/tree/master/src/proto/grpc/status

**Comment by fenos** [comment]
@seime Thank you for the link and for reporting this.

I believe that adding this specific type support to the client will make the experience better.
So i'll be including it in the next minor release

**Comment by fenos** [comment]
Unfortunately, I have to re-open this issue, as the current implementation which has been merged doesn't work when the app compiles, it just work on development mode 🤕 

The reason is that it tries to look for the imported protofile in the filesystem but the app is packaged up and not able to locate it.

This is a "bug" or "limitation" of protobuf.js.

I've opened an issue in protobuf.js repo here https://github.com/dcodeIO/protobuf.js/issues/1157
to allow me to proper implement this feature.

The fix will follow along once i know more 😄
